compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/UIKitView.uikit.kt

```diff
@@ -116,8 +117,8 @@ fun <T : UIView> UIKitView(
         }
     )
 
-    DisposableEffect(factory, onRelease) {
-        componentInfo.component = factory()
+    DisposableEffect(Unit) {
+        componentInfo.component = view
```

Comment:
We can write `componentInfo.component = factory()` here.

Reply:
Done